etl/steps/data/garden/demography/2023-04-07/population_education_wittgenstein.meta.yml

```diff
@@ -0,0 +1,37 @@
+# (Inherited from meadow, remove if not different.)
+all_sources:
+  - sources: &sources
+      name: Wittgenstein Centre (2018)
+      published_by: Wittgenstein Centre for Demography and Global Human Capital (WIC) Wittgenstein Centre Data Explorer. Version 2.0 2018
+      url: http://dataexplorer.wittgensteincentre.org/wcde-v2/
+      date_accessed: 2023-04-07
+      publication_year: 2018
+      # description: Source description.
+
+# (Inherited from meadow, remove if not different.)
+dataset:
+  sources:
+    - *sources
+
+tables:
+  population_education_wittgenstein:
+    # (Inherited from meadow, remove if not different.)
+    variables:
+      population:
+        title: Population
+        unit: persons
+        description: |
+          Population projections by levels of educational attainment.
+
+          • Under 15: Population below 15 years old
+          • No Education: Highest level of education attained: No level and Grade 1 of ISCED 1 not completed.
+          • Incomplete Primary: Highest level of education attained: Incomplete ISCED 1
+          • Primary: Highest level of education attained: Completed ISCED 1 and incomplete ISCED 2
+          • Lower Secondary: Highest level of education attained: Completed ISCED 2 and incomplete ISCED 3
+          • Upper Secondary: Highest level of education attained: Completed ISCED 3 and incomplete ISCED 4 5 or 6
+          • Post Secondary: Highest level of education attained: Completed ISCED 4 5 6 7 or 8
+          • Short Post Secondary: Highest level of education attained: Completed ISCED 4 or 5
+          • Bachelor: Highest level of education attained: Completed ISCED 6 and incomplete ISCED 7
+          • Master and higher: Highest level of education attained: Completed ISCED 7 or 8
+        sources:
```

Comment:
If sources are identical to dataset sources, then I think you don't need to specify them here.